### DISCUSSION TITLE
Feature Request: Easier way to choose colours for buttons and sliders (like mushroom card)

### DISCUSSION THREAD

**Discussion by staticlama** [Clooos/Bubble-Card#321] [Feature requests]
I've recently started using bubble card and I really like it. The main feature I'm missing (compared to mushroom) is the easy way you can override colour choices. For example, I have a selection of lights that are dimmable and non-dimmable, as well as other devices. In mushroom cards you can select one colour and it automatically adjusts the background colour, icon colour and the round background behind the icon.

To do this in bubble card requires a lot of configuration using styles, and I can't seem to get the "switch" style similar to the "slider" style.  For example see attached (top section is bubble card - bottom section is mushroom card)

![image](https://github.com/Clooos/Bubble-Card/assets/7868792/06c9518c-86a9-47fc-b29a-cabd859eb4fb)

Something similar to the mushroom card colour picker would be great. 
![image](https://github.com/Clooos/Bubble-Card/assets/7868792/7bc77776-ea82-4d99-948c-0226e96507ed)

**Comment by nonamer66** [comment]
I would appreciate this too. I'm not a fan of switches and sliders having different colors because the switch recognizes the color temperature where the slider does not.

**Comment by gabriel-stackhouse** [comment]
I third this request.  One of the greatest things about bubble-card is how easy it is to get beautiful and functional dashboards up-and-running.  Adding a color picker for setting colors would be another great step in that direction.
